app/views/users/edit_profile.html.erb

```diff
@@ -205,17 +205,17 @@
 
                 <div class="field margin-y-3 grid-col-9">
                   <%= f.label :bio, class: 'margin-0 margin-top-2' do %>
-                    <span>Bio</span>&nbsp;<span class="text-base-light profile-bio-character-count" id="user_<%= @user.id %>_profile_bio_character_count">(0/50 characters)</span>
+                    <span>Bio</span>&nbsp;
                     <p>Tell other users about yourself and your professional background.</p>
                   <% end %>
                   <%= f.text_area :bio, class: 'usa-input height-200px profile-bio border-base-light' %>
                 </div>
 
                 <% if is_fellow %>
                   <div class="field margin-y-3 grid-col-9">
-                    <%= f.label :credentials, 'Credentials (Public Bio)', class: 'margin-y-2' %><br/>
+                    <%= f.label :alt_job_title, 'Job Title (Public Bio)', class: 'margin-y-2' %><br/>
```

Comment:
Remove the `Enter as a comma-separated list, entries will list vertically on bio page` guidance on the line below